### PR TITLE
2. Set row col

### PR DESCRIPTION
**Pull request by thattil**
set custom row and column in the udp header. 

based on ctb_trasnsceiver_ro PR (steps: merge that, developer to this PR, visually easier PR then)

### CODE REVIEW

File: slsDetectorServers/ctbDetectorServer/slsDetectorFunctionList.c [hunk 1]
```diff
@@ -2605,8 +2605,8 @@ int readSample(int ns) {
             // if channel is in enable mask
             if ((1 << ich) & (transceiverMask)) {
 
-                int offset = FIFO_TIN_STATUS_FIFO_EMPTY_1_OFST + ich;
-                uint32_t mask = (1 << offset);
+                // int offset = FIFO_TIN_STATUS_FIFO_EMPTY_1_OFST + ich;
```
Comment (erikfrojdh) [inline slsDetectorServers/ctbDetectorServer/slsDetectorFunctionList.c:2608]: What is happening here?

File: slsDetectorServers/slsDetectorServer/src/slsDetectorServer_funcs.c [hunk 2]
```diff
@@ -10581,4 +10586,128 @@ int get_transceiver_enable(int file_des) {
...
+    return Server_SendResult(file_des, INT32, NULL, 0);
+}
+
+int get_column(int file_des) {
+    ret = OK;
+    memset(mess, 0, sizeof(mess));
+    int retval = -1;
+
+    LOG(logDEBUG1, ("Getting column\n"));
+    // get only
```
Comment (erikfrojdh) [inline slsDetectorServers/slsDetectorServer/src/slsDetectorServer_funcs.c:10644]: Do we need the comment? Get should only do get... I hope?

**Review by erikfrojdh — APPROVED** [review]

File: slsDetectorServers/ctbDetectorServer/slsDetectorFunctionList.c (hunk 1 above)
Comment (thattil) [reply slsDetectorServers/ctbDetectorServer/slsDetectorFunctionList.c:2608]: every transceiver channel (4 in total) have a bit to represent if its fifo is empty or not.

File: slsDetectorServers/slsDetectorServer/src/slsDetectorServer_funcs.c (hunk 2 above)
Comment (thattil) [reply slsDetectorServers/slsDetectorServer/src/slsDetectorServer_funcs.c:10644]: there are around 90 such comments and more that come from the time we rewrote the server client interface from scratch. it was more significant at the time. Maybe another PR to get rid of all these unnecessary commetns.

**Pull request merged**